openinfra_core/src/main/java/de/btu/openinfra/backend/rest/csv/TopicCsvResource.java

```diff
@@ -45,15 +63,27 @@ public Response getTopicAsCsv(
 			@Context UriInfo uriInfo,
 			@Context HttpServletResponse servletResponse) {
 		
-		// Remove the CSV extension of the URL and add query parameters
-		String path = uriInfo.getAbsolutePath().toString();
-		Client client = ClientBuilder.newClient();
-		WebTarget target = client.target(
-				path.substring(0, path.lastIndexOf(".csv")) + 
-				"?language=" + language);
-		String xml = target.request(
-				MediaType.APPLICATION_XML).get(String.class);
+		// Don't care about accessing the RBAC system classes. Just use the 

```

Comment:
Does a direct call of this URI allow access to the CSV if the corresponding topic characteristic has a read protection for the current user?